eng/pipelines/templates/steps/analyze.yml

```diff
@@ -311,13 +307,9 @@ steps:
 
   - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
 
-  - task: PowerShell@2
+  - pwsh: |
+      ./eng/scripts/Invoke-SmokeTests.ps1 "$(ChangedServices)"
     displayName: 'Run Nightly SmokeTests'
-    inputs:
-      targetType: 'filePath'
-      filePath: ./eng/scripts/Invoke-SmokeTests.ps1
-      pwsh: true
-      arguments: '$(ChangedServices)'
```

Comment:
This was converting "blah,blah2" to `[blah, blah2]` under the hood, which was messing with the filters. Just need it to pass as a string!

Reply:
Hmmm. Maybe it's because of the single quotes instead of the double, but I am confident this works so let's just go with this for now.